modules/overview-of-user-types-and-permissions.adoc

```diff
@@ -51,7 +51,7 @@ endif::[]
 
 Optionally, if you want to restrict access to your {productname-short} deployment, you can create specialized user groups for users and administrators.
 
-If you decide to restrict access, and you already have user groups defined in your configured identity provider, you can add these user groups to your {productname-short} deployment. If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {productname-short} and then add the appropriate users to them.
+If you decide to restrict access, and you already have user groups defined in your configured identity provider, you can add these user groups to your {productname-short} deployment. If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {openshift-platform} and then add the appropriate users to them.
```

Comment:
Could shorten this to just:

If you decide to use specialized user groups without adding these groups from an identity provider, you must create the groups in {openshift-platform} and then add users to them.